plugins/azure/fix_plugin_azure/utils.py

```diff
@@ -44,6 +44,26 @@ def case_insensitive_eq(left: T, right: T) -> bool:
         return left == right
 
 
+def from_str_to_typed(config_type: str, value: str) -> Any:
+    def set_bool(val: str) -> bool:
+        if val.lower() == "on":
```

Comment:
Have you checked that this is the only value for true? 
I can think of "yes", "true", "on", etc.

Reply:
I listed all enumeration values and if the value was boolean, it was either "on" or "off". Same for `postgresql` and `mysql` services